tests/source/issue-3876.rs

```diff
@@ -0,0 +1,29 @@
+fn f<S>(s: S)
+where
+    S: Send /* */,
+      //
+{
+}
+
+fn main() {
+    foo(
+        x 
+	// comment 1
+        ,
+        // comment 2
+    )
+}
+
+fn main() {
+    foo(
+        x
+        // comment 1
+
+
+        ,
+
+
+
+        // comment 2
+    )
+}
```

Comment:
Could you also add some test cases that include multi-line block style comments?